tests/integration/tier1/kernel-boot-files/test_verify_file_presence.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+
+from envparse import env
+
+
+@pytest.mark.check_boot_files_presence
+def test_check_boot_files_presence(convert2rhel):
+    """
+    Verify that the conversion was successfull and the kernel boot files are
+    present.
+
+    Our criteria of a successfull conversion, in this case, is related to the
+    `check_kernel_boot_files()` finding the two necessary boot files (initramfs
+    and vmlinuz), and validating that the initramfs file is not corrupted.
+    """
+
+    with convert2rhel(
+        "-y --no-rpm-va --serverurl {} --username {} --password {} --pool {} --debug".format(
+            env.str("RHSM_SERVER_URL"),
+            env.str("RHSM_USERNAME"),
+            env.str("RHSM_PASSWORD"),
+            env.str("RHSM_POOL"),
+        )
+    ) as c2r:
+        assert c2r.expect("Checking if vmlinuz file exists on the system.") == 0
+        assert c2r.expect("Checking if initiramfs file exists on the system.") == 0
+        assert c2r.expect("Initramfs and vmlinuz files exists and are valid.") == 0
```

Comment:
Just a proposal.
Given that this is just a simple conversion with no significant preparation, I would say we can even scrap it as an individual test and run it as an after-conversion-check.
Running this simple conversion could (IMHO) unnecessarily prolong the pipeline execution time by one full conversion runtime.
Maybe a good first use case for the check going over the log file matching required strings we were talking about @r0x0d .
If you would agree with this, I can provide you with a syntax, to skip the check for the test cases with tainted results.
WDYT @r0x0d, @SpyTec, @bocekm ?

Reply:
Agreed

Reply:
Sounds good.

Reply:
Thanks, implemented